SonarQube.Common/AnalysisProperties/AnalysisProperties.cs

```diff
@@ -68,9 +69,15 @@ public static AnalysisProperties Load(string fileName)
                 throw new ArgumentNullException("fileName");
             }
 
-            var model = Serializer.LoadModel<AnalysisProperties>(fileName);
```

Comment:
I have renamed the variable because it was too weird to have a singular name for a collection.